api/src/test/java/org/openmrs/util/databasechange/Database1_9_7UpgradeIT.java

```diff
@@ -64,6 +66,12 @@
 	
 	private DatabaseUpgradeTestUtil upgradeTestUtil;
 	
+	/**
+	 * Create a temporary folder for the application directory datat
+	 */
+	@ClassRule
+	public static TemporaryFolder TEMP_APP_DIR = TemporaryFolder.builder().assureDeletion().build();
+	
 	private static File testAppDataDir;
 	
 	
```

Comment:
I think you are missing the same changes as in the other 2 files. Rember we now only need to SRT runtime properties and the global property